docs/fast_data/projection_storer.md

```diff
@@ -0,0 +1,53 @@
+---
+id: projection_storer
+title: Projection Storer
+sidebar_label: Projection Storer
+---
+
+Projection Storer is one of the Fast Data components, which it acts as an entrypoint to the system. Its role is to ingest change events coming from the
+origin system ([System of Records](/fast_data/the_basics.md#system-of-records-sor)) and process them by cleaning, filtering,
+validating and eventually storing them as raw data for the system.
+
+The service consumes Kafka messages as [_ingestion events_](/fast_data/inputs_and_outputs.md#ingestion-message) and for each
+input change event produces as output the corresponding [_projection record update event_](/fast_data/inputs_and_outputs.md#projection-update-message).
+
+:::note
+Projection Storer service generates _projection record update events_ that adopt the `v2.0.0` schema, which differ slightly
+from the events generated by the [Real-Time Updater](/fast_data/realtime_updater.md) service, which uses the `v1.0.0` schema. Nonetheless, both event schemas are
+supported, recognized and handled correctly by downstream components.
+:::
+
+### Comparison with Real-Time Updater
+
+Projection Storer features partially overlap with the ones of [Real-Time Updater](/fast_data/realtime_updater.md) service. In fact,
+Projection Storer implements a comparable processing logic for consuming ingestion events, storing projection records and notifying
+changes via projection record update events.  
+When Fast Data is configured adopting [Event-Driven architecture](/fast_data/architecture.md#event-driven-architecture),
+the Projection Storer can be employed as a drop-in replacement of Real-Time Updater service.
```

Comment:
It could be worth adding why the PS is better, like performance improvements and easier configuration, just so the user knows why should he replace the old one and jump into the configuration page